src/parsing.jl

```diff
@@ -279,7 +279,9 @@ function pqparse(::Type{Time}, str::AbstractString)
 end
 
 # InfExtendedTime support for Dates.TimeType
-function pqparse(::Type{InfExtendedTime{T}}, str::AbstractString) where {T<:Dates.TimeType}
+function pqparse(
+    ::Type{InfExtendedTime{T}}, str::AbstractString,
+) where {T <: Dates.TimeType}
```

Comment:
This space around `<:` caused the line length to grow past the limit; were we going to remove this space?

Reply:
i prefer `whitespace_typedefs=false`, but i ran on this code-base with `whitespace_typedefs=true` because that was the far more common pattern.

We could instead run with `whitespace_typedefs=false` (xref https://github.com/invenia/BlueStyle/issues/70)